### PR TITLE
Update template

### PR DESCRIPTION
**Pull request by daun**
- Update base plugin version
- Use lint and build commands of the base plugin
- Add peer dependencies
- Update readme

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,23 +1,25 @@
 import Plugin from '@swup/plugin';
 
-export default class PluginName extends Plugin {
-	name = 'PluginName';
+export default class SwupPluginNamePlugin extends Plugin {
```
Comment (gmrchk) [inline src/index.js]: I'm not sure if you're aware, but maybe you got it covered already and the new strings are put in here on purpose. All of these strings (PluginName, [PluginName], etc.) are [used by CLI](https://github.com/swup/cli/blob/master/src/commands/create.js#L98-L108) to replace it with the correct format when creating a new plugin.

Comment (daun) [reply src/index.js]: Good to know, wasn't aware of that. I'll revert those and test this new version with the CLI.

Comment (daun) [reply src/index.js]: I've reverted all changes to name placeholders.

File: .gitignore [hunk 2]
```diff
@@ -13,10 +13,4 @@ wiki-wishlist
 *.sublime-workspace
 .editorconfig
 .idea
-lib
-/plugins
-.vscode/*
-!.vscode/settings.json
```
Comment (hirasso) [inline .gitignore:19]: I would keep all `.vscode` entries in `.gitignore`.

Comment (hirasso) [reply .gitignore:19]: ...or only `.vscode`, of course

Comment (daun) [reply .gitignore:19]: Guess that just happened. Feel free to change it back!  I won't have time to edit anything about the PR for the next few days.

**Review by hirasso — DISMISSED** [review]

**Comment by daun** [comment]
@hirasso can we remove the package lock again? it's just a template repo and not meant to be installed.

**Comment by hirasso** [comment]
yes :)

**Review by hirasso — APPROVED** [review]

**Pull request merged**